libtransmission/announcer.c

```diff
@@ -153,6 +181,32 @@ typedef struct tr_announcer
 }
 tr_announcer;
 
+static struct tr_scrape_info* tr_announcerGetScrapeInfo(struct tr_announcer* announcer, const char* url)
+{
+    struct tr_scrape_info* info = NULL;
+
+    if (url && *url)
```

Comment:
Here and everywhere: There was another commit at some point that made sure that all conditional expressions are explicitly boolean. One would then prefer writing this instead:
```suggestion
    if (url != NULL && *url != '\0')
```

Reply:
Changed 88198468abe1684558dd9cd7f576d8e9fa82f315.

NB, you mentioned "here and everywhere" but I didn't see another instance of this. If I've missed something please let me know

Reply:
Good lord they _were_ everywhere. :sweat_smile: